config/locales/en.yml

```diff
@@ -225,6 +225,9 @@ en:
       transparency:
         one: Transparency data
         other: Transparency data
+      world_location_news_article:
```

Comment:
Had a quick chat in the core channel about this and one suggestion from @boffbowsh is that eventually WLNA might be consolidated into being a "news_article". This sounds reasonable but I'm not sure what's involved in doing this though. It possibly needs some investigation (created trello card for this: https://trello.com/c/kAJM8OfE/815-investigate-consolidating-wlna-and-news-article) and if we do want to do that then perhaps it warrants a story of its own.

For now, I've kept the entry in `en.yml` so that history notices show for WLNA. In addition, I've added empty entries to the other locales for world_location_news_article with commit 8ec75a8fb59f22e8ac9e36a596b18e5b1cc71393 so it's consistent with the rest of the application - there are other empty entries in the locales (thanks @gpeng) as well so following prior art in the meantime.